<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecisionsModelSelectorWidget</class>
 <widget class="QWidget" name="DecisionsModelSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Decisions model scheme</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>132</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="userSettingsGroupBox">
     <property name="title">
      <string>User settings</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="decisionsModelComboBox"/>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="enableAllSchemesCheckBox">
     <property name="text">
      <string>Enable all</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
